internal/cleanup/cleanup.go

```diff
@@ -81,25 +77,25 @@ func (s *AuthenticationService) Run(ctx context.Context) {
 						acquiredUntil = now.Add(minCleanupInterval)
 					}
 
-					s.logger.Warn("attempting to acquire lock", zap.Error(err))
+					logger.Warn("attempting to acquire lock", zap.Error(err))
 					continue
 				}
 
 				// update the next sleep target to current entries acquired until
 				acquiredUntil = entry.AcquiredUntil
 
 				if !acquired {
-					s.logger.Info("cleanup process not acquired", zap.Time("next_attempt", entry.AcquiredUntil))
+					logger.Info("cleanup process not acquired", zap.Time("next_attempt", entry.AcquiredUntil))
```

Comment:
same, trying to decide what constitutes a `debug` message and what should be `info`